docs/design/admission_control_limit_range.md

```diff
@@ -73,8 +73,8 @@ type LimitRangeItem struct {
   Min ResourceList `json:"min,omitempty" description:"min usage constraints on this kind by resource name"`
   // Default resource limits on this kind by resource name
   Default ResourceList `json:"default,omitempty" description:"default resource limits values on this kind by resource name if omitted"`
-  // DefaultRequests resource requests on this kind by resource name
-  DefaultRequests ResourceList `json:"defaultRequests,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
+  // DefaultRequest resource requests on this kind by resource name
+  DefaultRequest ResourceList `json:"defaultRequest,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
```

Comment:
s/if omitted/if resource request is omitted/

For LimitRequestRatio comment on next line it is clearer if you say "LimitRequestRatio is the maximum allows ratio of limit over request, representing the maximum burst for the named resource" or something like that. As-is, it's confusing.